src/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFile.Extract.cs

```diff
@@ -74,9 +74,9 @@ public static partial class ZipFile
         /// 
         /// <param name="sourceArchiveFileName">The path to the archive on the file system that is to be extracted.</param>
         /// <param name="destinationDirectoryName">The path to the directory on the file system. The directory specified must not exist, but the directory that it is contained in must exist.</param>
-        /// <param name="overwrite">True to indicate overwrite.</param> 
-        public static void ExtractToDirectory(string sourceArchiveFileName, string destinationDirectoryName, bool overwrite) => 
-            ExtractToDirectory(sourceArchiveFileName, destinationDirectoryName, entryNameEncoding: null, overwrite: overwrite);
+        /// <param name="overWriteFiles">True to indicate overwrite.</param> 
+        public static void ExtractToDirectory(string sourceArchiveFileName, string destinationDirectoryName, bool overWriteFiles) => 
```

Comment:
`overwriteFiles`, right? (not a capital-W)

Reply:
Yes, will fix. Good catch :)